### DISCUSSION TITLE
How to get search results into spreadsheet

### DISCUSSION THREAD

**Discussion by Liturgist** [community/community#52272] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Once a search result is produced, current or beta, how can I get it into a spreadsheet?

Can the current or beta code search be scripted with GitHub CLI or any other way?  <<< NEEDED

**Comment by look** [comment, marked as answer]
We are working on an API for the new search, but it is not yet available. That will be the best way to get results into a spreadsheet.

Can you give some more details about your use case to help our development? Thank you.
